### ISSUE TITLE
Dub example not compiling

### ISSUE THREAD

**Issue by nicesai** [vibe-d/vibe.d#2243]
Screenshot here: 
https://imgur.com/a/ySPSmQS

I am trying to compile default example that comes with dub. When I created the dub package using

dub --init web_test -t vibe.d

here is that sample code:

import vibe.vibe;

void main()
{
	auto settings = new HTTPServerSettings;
	settings.port = 8080;
	settings.bindAddresses = ["::1", "127.0.0.1"];
	listenHTTP(settings, &hello);

	logInfo("Please open http://127.0.0.1:8080/ in your browser.");
	runApplication();
}

void hello(HTTPServerRequest req, HTTPServerResponse res)
{
	res.writeBody("Hello, World!");
}

**Comment by oconnor0** [comment]
I was running into the same issue and asked at https://forum.dlang.org/thread/bobqtoesasjlwvdiisih@forum.dlang.org I was told to try: `dub --arch=x86_64` which worked.

**Comment by Beyarz** [comment]
Yes, you have to define which architecture you are running.

**Comment by oconnor0** [comment]
That information would be good to add to https://vibed.org/blog/posts/a-scalable-chat-room-service-in-d

**Comment by nicesai** [comment]
When I try "dub --arch=x86_64" I got this error:

<snip>
vibe-d:redis 0.8.4: building configuration "library"...
vibe-d:web 0.8.4: building configuration "library"...
vibe-d 0.8.4: building configuration "vibe-core"...
webt ~master: building configuration "application"...
Linking...
libcmt.lib(chandler.obj) : error LNK2019: unresolved external symbol __imp_RtlUn
windEx referenced in function __C_specific_handler
libcmt.lib(invarg.obj) : error LNK2019: unresolved external symbol RtlVirtualUnw
ind referenced in function _call_reportfault
libcmt.lib(gs_report.obj) : error LNK2001: unresolved external symbol RtlVirtual
Unwind
libcmt.lib(invarg.obj) : error LNK2019: unresolved external symbol RtlLookupFunc
tionEntry referenced in function _call_reportfault
libcmt.lib(gs_report.obj) : error LNK2001: unresolved external symbol RtlLookupF
unctionEntry
libcmt.lib(jmpuwind.obj) : error LNK2019: unresolved external symbol RtlUnwindEx
 referenced in function _local_unwind
.dub\build\application-debug-windows-x86_64-dmd_2083-75C7BA8F6461BA35B2AC6FD6A7C
CB14D\webt.exe : fatal error LNK1120: 4 unresolved externals
Error: linker exited with status 1120
C:\xxxxxxx\D\dmd2\windows\bin\dmd.exe failed with exit code 1.


when I tried with "dub --arch=x86" I got the same exception as in the original post.


Any suggestions?
